src/browser-frontend/views/browser/chrome/tabbar/tab.css

```diff
@@ -24,27 +32,37 @@ specific language governing permissions and limitations under the License.
 }
 
 .tab.selected {
-  background: var(--theme-tab-active-background);
+  background-position: var(--theme-tab-active-background-position);
+  box-shadow: var(--theme-focus-outline-style);
   color: var(--theme-tab-active-color);
 }
 
 .tab.deselected {
-  background: var(--theme-tab-inactive-background);
-  box-shadow: 1px 0 0 0 var(--theme-tab-separator-color);
-  color: var(--theme-tab-inactive-color);
+  /* react-css-modules generateAppendClassName requires that this exist */
 }
 
 .tab.initial {
   width: var(--theme-tab-initial-width);
+  height: var(--theme-tab-initial-height);
+  background-position: var(--theme-tab-inactive-background-position);
   opacity: var(--theme-tab-initial-opacity);
 }
 
 .tab.open {
-  width: var(--theme-tab-open-width);
+  height: var(--theme-tab-open-height);
   opacity: var(--theme-tab-open-opacity);
 }
 
 .tab.closed {
   width: var(--theme-tab-closed-width);
   opacity: var(--theme-tab-closed-opacity);
+  transition: width 0.5s var(--animation-curve);
+}
+
+.tab.selected-closed {
+  width: var(--theme-tab-closed-width);
+  height: var(--theme-tab-closed-height);
+  background-position: var(--theme-tab-active-background-position);
```

Comment:
I can't for the life of me figure out what this `background-position` in this `.tab.selected-closed` selector actually does, visibly. I've made the animations 20x longer, removed it, and don't see any difference at all.

Looking more closely, I notice that deselected hovered tabs overwrite backgrounds anyway, so this rule won't have any visible effect unless we exclude `.selected-closed` from the `.tab:not(.selected):hover` rule.